wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,303 @@
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 | October 2007 |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 | 2007-10-11 |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-12-24 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 | *2008-09-07* |
+
+## 2008
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-10-07 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 | 2008-12-11 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 | *2009-01-19* |
+
+## 2009
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | *2008-12-11* | 2009-01-19 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 | March 2009 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 | March 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | March 2009 | April 2009 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | April 2009 | April 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | April 2009 | June 2009 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | July 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | July 2009 | July 2009 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | September 2009 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | September 2009 | November 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | *2010-02-07* |
+
+## 2010
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *November 2009* | 2010-02-07 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 | 2010-09-08 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-09-08 | September 2010 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 | October 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 | November 2010 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 | November 2010 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | December 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | December 2010 | *January 2011* |
+
+## 2011
+
+| Player | From | To |
+| --: | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *December 2010* | January 2011 |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | January 2011 | January 2011 |
```

Comment:
```suggestion
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2011-01-09 | January 2011 |
```
As per https://github.com/ppy/osu-wiki/pull/4720/files#r551291399

Reply:
Updated with https://github.com/ppy/osu-wiki/pull/4720/commits/b9f1ab80d6cb9c10aaabdd4b370ffe02008be4b8, need to add links to sources column at somepoint